x/incentive/types/params.go

```diff
@@ -207,6 +207,11 @@ func (rp RewardPeriod) Validate() error {
 	if !rp.RewardsPerSecond.IsValid() {
 		return fmt.Errorf("invalid reward amount: %s", rp.RewardsPerSecond)
 	}
+
+	if rp.RewardsPerSecond.Amount.IsZero() {
+		return fmt.Errorf("reward amount cannot be zero: %v", rp.RewardsPerSecond)
+	}
+
```

Comment:
nit: what was the reason for this change?

Reply:
It is passed to `sdk.NewDecCoinsFromCoins` in the accumulator which would have the sdk bug in the method if it has a 0 amount